.ci/docker/build.sh

```diff
@@ -37,6 +37,10 @@ case "${IMAGE_NAME}" in
     ARM_SDK=yes
     CLANG_VERSION=12
     ;;
+  executorch-ubuntu-22.04-qnn-sdk)
```

Comment:
Why not adding qnn sdk by extending the `executorch-ubuntu-22.04-clang12-android` image below? QNN backend would require ndk and cross-compilation, making it a good fit to the android image

Reply:
this is not android, it's x86